Tabloid/Controllers/HelloController.cs

```diff
@@ -3,6 +3,7 @@
 
 namespace Tabloid.Controllers
 {
+    [Authorize]
     [Route("api/[controller]")]
```

Comment:
You don't need to add the [Authorize] attribute here. This is a test controller that has two endpoints, one to test without auth and another one to test with auth. Adding the attribute to the whole class would defeat the purpose of that.

Reply:
removed it. thanks